File: Pattern 15/Pattern_15.cpp
class Solution {
  public:
    void printTriangle(int n) {
        
        // just reverse version of previous question
        // iterate on outer loop on backward
        for(int row = n; row > 0; row--) {
            char no = 'A';
            for(int col =1; col<=row; col++) {
                cout<<no++;
            }
            cout<<endl;
        }
    }
};

File: Pattern 15/Pattern_15.py
class Solution:
    def printTriangle(self, n):
        # just reverse version of previous question
        # iterate on outer loop on backward
        for row in range(n,0,-1):
            no = 65
            for col in range(1,row+1):
                print(chr(no),end = "")
                no +=1
            print()
        
